crates/wasi-provider/src/lib.rs

```diff
@@ -59,21 +59,13 @@ impl Provider for WasiProvider {
         // and then execute the WASM, passing in the relevant data.
         // When the pod finishes, we update the status to Succeeded unless it
         // produces an error, in which case we mark it Failed.
-        debug!(
-            "Pod added {:?}",
-            pod.metadata.as_ref().and_then(|m| m.name.as_ref())
-        );
         let namespace = pod
             .metadata
             .as_ref()
             .and_then(|m| m.namespace.as_deref())
             .unwrap_or_else(|| "default");
 
         // TODO: Implement this for real.
-        // Okay, so here is where things are REALLY unfinished. Right now, we are
-        // only running the first container in a pod. And we are not using the
-        // init containers at all. And they are not executed on their own threads.
```

Comment:
From what I can read, the comment about init containers comment still rings true. We still don't fire off any init containers. Should we keep that part of the comment in here (and in wascc-provider) so we know we still need to implement this?

Reply:
This is captured in #25 so I thought it wasn't necessary here any more

Reply:
Great! As long as we got that captured then I'm happy. :)